.github/actions/usd-install-dep/action.yml

```diff
@@ -50,13 +52,14 @@ runs:
         -DPXR_BUILD_ALEMBIC_PLUGIN:BOOL=OFF
         -DPXR_BUILD_EMBREE_PLUGIN:BOOL=OFF
         -DPXR_BUILD_IMAGING:BOOL=OFF
-        -DPXR_BUILD_MONOLITHIC:BOOL=OFF
+        -DPXR_BUILD_MONOLITHIC:BOOL=ON
         -DPXR_BUILD_TESTS:BOOL=OFF
         -DPXR_BUILD_USD_IMAGING:BOOL=OFF
         -DPXR_ENABLE_PYTHON_SUPPORT:BOOL=OFF
         -DPXR_BUILD_EXAMPLES:BOOL=OFF
         -DPXR_BUILD_TUTORIALS:BOOL=OFF
         -DPXR_BUILD_USD_TOOLS:BOOL=OFF
+        -DPXR_INSTALL_LOCATION:PATH=../lib/usd
```

Comment:
what is this relative to ?

Reply:
Could be a USD shared library or any binary embedding it, so that means `bin/` or `lib/`.

Reply:
I'm confused. I dont understand what does PXR_INSTALL_LOCATION means and why do you use a relative path here.

Reply:
the path is used to locate the `plugInfo.json` file, located in `<install>/lib/usd`.  
By default, it looks in `./usd` which work fine if the binary is located in `<prefix>/lib`, but not if the binary is in `<prefix>/bin` (on Windows)

Reply:
ok got it.